packages/react-native/Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -129,20 +130,32 @@ class KeyboardAvoidingView extends React.Component<Props, State> {
     }
   };
 
+  // Avoid unnecessary renders if the KeyboardAvoidingView is disabled.
+  _setBottom = (value: number) => {
+    const {enabled = true} = this.props;
+    this._bottom = value;
+    if (enabled === true) {
```

Comment:
```suggestion
    if (enabled) {
```
Self explanatory, though stylistic :)

Reply:
@adamgrzybowski  I see you made that change on your 2nd commit.  What was your thinking?

Reply:
@blakef @Pranav-yadav done